### PR TITLE
Generate Documentation and Development links into the README.md when …

### LINKED ISSUE #1398: dev_url and doc_url are not used to generated feedstock README.md
The meta dev_url and doc_url are not used when generating the feedstock README.md

### PR DESCRIPTION
**Pull request by jwillemsen**
Closes #1398

### CODE REVIEW

**Comment by scopatz** [comment]
Please add a news entry: https://regro.github.io/rever-docs/news.html

**Comment by scopatz** [comment]
Thanks!

File: conda_smithy/templates/README.md.tmpl [hunk 1]
```diff
@@ -14,6 +14,10 @@ Feedstock license: BSD-3-Clause
 
 Summary: {{ package.meta.about.summary }}
 
+{% if package.meta.about.dev_url %}Development: {{ package.meta.about.dev_url }}{% endif %}
```
Comment (isuruf) [inline conda_smithy/templates/README.md.tmpl:17]: Isn't this adding unnecessary extra spaces when `dev_url` is not in the meta?

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:17]: Probably what do we need to fix that. This?

```suggestion
{% if package.meta.about.dev_url %}Development: {{ package.meta.about.dev_url }}{%- endif %}
```

Comment (jwillemsen) [reply conda_smithy/templates/README.md.tmpl:17]: My change did add multiple new lines when no content was defined, so created https://github.com/conda-forge/conda-smithy/pull/1400

Comment (jakirkham) [reply conda_smithy/templates/README.md.tmpl:17]: Thanks :)

**Pull request merged**